src/app/space/create/deployments/apps/deployment-card.component.spec.ts

```diff
@@ -73,67 +82,158 @@ class FakeDeploymentDetailsComponent {
   @Input() spaceId: string;
 }
 
-describe('DeploymentCardComponent', () => {
+function initMockSvc(mockSvc: jasmine.SpyObj<DeploymentsService>, active: Subject<boolean>) {
+  mockSvc.getVersion.and.returnValue(Observable.of('1.2.3'));
+  mockSvc.getCpuStat.and.returnValue(Observable.of({ used: 1, quota: 2 }));
+  mockSvc.getMemoryStat.and.returnValue(Observable.of({ used: 3, quota: 4, units: 'GB' }));
+  mockSvc.getAppUrl.and.returnValue(Observable.of('mockAppUrl'));
+  mockSvc.getConsoleUrl.and.returnValue(Observable.of('mockConsoleUrl'));
+  mockSvc.getLogsUrl.and.returnValue(Observable.of('mockLogsUrl'));
+  mockSvc.deleteApplication.and.returnValue(Observable.of('mockDeletedMessage'));
+  mockSvc.isApplicationDeployedInEnvironment.and.returnValue(active);
+}
+
+describe('DeploymentCardComponent async tests', () => {
+
+    let component: DeploymentCardComponent;
+    let fixture: ComponentFixture<DeploymentCardComponent>;
+    let mockSvc: jasmine.SpyObj<DeploymentsService>;
+    let notifications: any;
+    let active: Subject<boolean>;
+
+    beforeEach(fakeAsync(() => {
+      active = new BehaviorSubject<boolean>(true);
+      mockSvc = createMock(DeploymentsService);
+      initMockSvc(mockSvc, active);
+      notifications = jasmine.createSpyObj<NotificationsService>('NotificationsService', ['message']);
+
+      TestBed.configureTestingModule({
+        declarations: [
+          DeploymentCardComponent,
+          FakeDeploymentsDonutComponent,
+          FakeDeploymentGraphLabelComponent,
+          FakeDeploymentDetailsComponent,
+          FakeDeploymentStatusIconComponent
+        ],
+        imports: [ BsDropdownModule.forRoot(), CollapseModule.forRoot(), ChartModule ],
```

Comment:
Could you split this array with newlines so it matches the declarations and providers arrays?